wasmCloud/wasmCloud
# build(deps): added missing dependencies to washboard-ui

## Feature or Problem

`radix-ui/react-collapsible` and `react-router` are used in the front end but didn't appear in `package.json`. This means they weren't installed alongside other dependencies, causing an error in the terminal and the browser to hang when trying to load the UI. This PR hotfixes that by adding them to the package list so that the UI renders.

## Related Issues

https://github.com/wasmCloud/wasmCloud/issues/962

## Release Information

next

## Consumer Impact

See details above, no steps are needed.

## Testing

n/a

### Unit Test(s)
 
n/a

### Acceptance or Integration

n/a

### Manual Verification

1. Clone repo.
2. cd to `/washboard-ui`.
3. Run `npm run dev`.
4. Check the terminal for error messages.
5. Navigate to `localhost:5173` in a browser.
